app/assets/javascripts/utilities/userData.js

```diff
@@ -1,12 +1,7 @@
 'use strict';
 
 function userData() {
-  const dataUser = document
-    .getElementsByTagName('body')[0]
-    .getAttribute('data-user');
+  const { user = null } = document.body.dataset;
```

Comment:
For those curious about dataset, here is the lowdown via MDN on [dataset](https://developer.mozilla.org/en-US/docs/Web/API/HTMLOrForeignElement/dataset)